### DISCUSSION TITLE
Transcoding audio to PCM, processing and writing back to another video: possible ?

### DISCUSSION THREAD

**Discussion by yv2149** [PyAV-Org/PyAV#868] [1. Help]
Hi,
I'm trying to postprocess audio from any  video source.
To do so, I intend to:
1. convert source audio frames (whatever codec), to raw PCM S16LE stereo.
2. postprocess my raw S16LE data 
3. encode the postprocesed  S16LE data to back Audio Frames as "wav" codec, in another video 

Is that somehow possible with PyAV?

Thanks in advance!

**Comment by mikeboers** [comment]
The stumbling block at this point _might_ be copying video data without transcoding it. I don't recall the status of that sort of thing... maybe it works for simple video.
